api-scanner/src/test/resources/kotlin-annotations/kotlin/net/corda/example/ClassWithFilteredMethods.kt

```diff
@@ -0,0 +1,5 @@
+@file:Suppress("UNUSED")
+package net.corda.example
+
+
+class ClassWithFilteredMethods @JvmOverloads constructor(val number: Int = 1, val text: String = "yummy")
```

Comment:
Technically speaking, this isn't a case where we would like to exclude the constructor from the API Scanner.